bluebrain/deployment/environments/applications_science.yaml

```diff
@@ -18,6 +18,7 @@ spack:
           - py-efel
           - py-synthesis-workflow
           - py-visual-cortex-analysis
+          - py-emodel-generalisation
```

Comment:
This only enables a module, but does not build the software. Please add the same entry after line 45 :)

Reply:
ah right, thanks!